docs/ibc/upgrades/quick-guide.md

```diff
@@ -0,0 +1,44 @@
+# How to Upgrade IBC Chains and their Clients
```

Comment:
I don't understand the difference with the first doc from this title

Reply:
This doc is intended for the end users of the upgrade process. Relayers and stake holders of the upgrading chain.